tests/unit/src/Internal/LoggerTest.php

```diff
@@ -0,0 +1,300 @@
+<?php
+/**
+ * Class LoggerTest
+ *
+ * @package WooCommerce\Payments
+ */
+
+namespace WCPay\Tests;
+
+use WCPAY_UnitTestCase;
+use WC_Logger;
+use WC_Log_Levels;
+use WC_Payment_Gateway_WCPay;
+use Exception;
+use WCPay\Core\Mode;
+use WCPay\Internal\Logger;
+
+/**
+ * Internal Logger tests.
+ *
+ */
+class LoggerTest extends WCPAY_UnitTestCase {
+	/**
+	 * Holds the logger, which will be tested.
+	 *
+	 * @var Logger
+	 */
+	private $logger;
```

Comment:
Let's call this `$sut` for consistency with other unit tests for `src`.

Reply:
Done